docs/diagrams/ModelClassDiagram.puml

```diff
@@ -5,52 +5,58 @@ skinparam arrowColor MODEL_COLOR
 skinparam classBackgroundColor MODEL_COLOR
 
 Package Model <<Rectangle>>{
-Interface ReadOnlyAddressBook <<Interface>>
+Interface ReadOnlyReeve <<Interface>>
 Interface Model <<Interface>>
 Interface ObservableList <<Interface>>
-Class AddressBook
-Class ReadOnlyAddressBook
+Class Reeve
+Class ReadOnlyReeve
 Class Model
 Class ModelManager
 Class UserPrefs
 Class ReadOnlyUserPrefs
 
-Package Person {
-Class Person
-Class Address
-Class Email
+Package Student {
+Class Student
+Class School
+Class Year
 Class Name
 Class Phone
-Class UniquePersonList
+Class UniqueStudentList
 }
 
-Package Tag {
-Class Tag
+Package Admin {
+Class Admin
+}
+
+Package Academic {
+Class Academic
```

Comment:
Admin and Academic are actually packages within Student, could you perhaps nest them within the Student package?

Reply:
oh yea thanks